client/ayon_houdini/plugins/publish/extract_last_published.py

```diff
@@ -0,0 +1,119 @@
+import os
+import shutil
+
+import clique
+import pyblish.api
+
+from ayon_core.lib import collect_frames
+from ayon_houdini.api import plugin
+
+
+class ExtractLastPublished(plugin.HoudiniExtractorPlugin):
+    """Extractor copying files from last published to staging directory.
+
+    It works only if instance data includes "last_version_published_files"
+    and there are frames to fix.
+
+    The files from last published are based on files which will be
+    extended/fixed for specific frames.
+
+    NOTE: 
+        This plugin is closely taken from ayon-nuke.
+        It contains some Houdini addon specific logic as various addons may
+          have unique methods for managing `staging_dir`, `expectedFiles`
+          and `frames`.
+    TODO:
+        It's preferable to to generalize this plugin for broader use and
+          integrate it into ayon-core.
+    """
+
+    order = pyblish.api.ExtractorOrder - 0.1
+    label = "Extract Last Published"
+    targets = ["local"]  # Same target as `CollectFramesFixDef`
+    families = ["*"]
+
+    def process(self, instance):
+        frames_to_fix = instance.data.get("frames_to_fix")
+        if not frames_to_fix:
+            self.log.debug("Skipping, No frames to fix.")
+            return
+
```

Comment:
I wonder if should we add this one here too.
```suggestion
        if not instance.data.get("integrate", True):
            self.log.debug("Skipping collecting frames to fix data for "
                           "instance because instance is set to not integrate")
            return
            
```

Reply:
I guess yes as you are not gonna extract the files and transfer them to the publish folder.

Reply:
well, yes. 
But it's also worth mentioning that instances with `integrate == False` don't usually have any last published files so I always get 
```
Skipping, No last publish found.
```

Reply:
https://github.com/ynput/ayon-houdini/pull/45/commits/c96e05415d6f939bfc8f2abad9b9ccdb6d1b254f